### ISSUE TITLE
Titles not showing up for screenreader on My Articles assignment

### ISSUE THREAD

**Issue by bwreid** [WikiEducationFoundation/WikiEduDashboard#3236]
### What is happening?

Screenreaders are unable to read the titles for articles when clicking on the "Review an Article" button. This is likely the same for the nearby "Assign myself an Article" button.

![image](https://user-images.githubusercontent.com/1316902/63813502-b8cb3500-c8e2-11e9-88d9-0c28f44cedb1.png)

### To Reproduce

1. Go to a page with available articles to review/assign
1. Using a screenreader, attempt to read the articles in the dropdown list

### Expected behavior

The titles should be read.

**Comment by MasterIfeanyi** [comment]
It works for me - I tested using NVDA, 2026. 

May we close this ? @ragesoss

**Comment by ragesoss** [comment]
Sure, thanks @MasterIfeanyi !
